x-pack/test/reporting/api/usage.ts

```diff
@@ -5,22 +5,29 @@
  */
 
 import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+import { ReportingUsageStats } from '../services/reporting_api';
 import * as GenerationUrls from './generation_urls';
 
-export default function({ getService }) {
+interface UsageStats {
+  reporting: ReportingUsageStats;
+}
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
   const reportingAPI = getService('reportingAPI');
-  const usageAPI = getService('usageAPI');
+  const usageAPI = getService('usageAPI' as any);
```

Comment:
Note: this service is mistakenly using `/api/stats?extended` and returning the data in a different format where all the fields are snake_cased.